test/performance/search/dream_index/binning_directory_benchmark.cpp

```diff
@@ -0,0 +1,144 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/search/dream_index/binning_directory.hpp>
+
+using namespace seqan3;
+
+inline benchmark::Counter hashes_per_second(size_t const count)
+{
+    return benchmark::Counter(count,
+                              benchmark::Counter::kIsIterationInvariantRate,
+                              benchmark::Counter::OneK::kIs1000);
+}
+
+static void arguments(benchmark::internal::Benchmark* b)
+{
+    for (int32_t bins : {64, 8192})
+    {
+        for (int32_t bits = 1<<15; bits <= 1<<25; bits <<= 5)
+        {
+            for (int32_t hash_num = 2; hash_num < 4; ++hash_num)
+            {
+                b->Args({bins, bits, hash_num, 100'000});
+            }
+        }
+    }
+}
+
+inline std::vector<size_t> generate_numbers(size_t const len = 500,
```

Comment:
You can use the `generate_sequence` function in `test/include/...`. It also works with numbers